platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineManager.java

```diff
@@ -149,15 +149,16 @@ private Handler getHandler() {
    *
    * @param callback the callback to be invoked
    */
-  public void listOfflineRegions(@NonNull
-                                 final ListOfflineRegionsCallback callback) {
+  public void listOfflineRegions(@NonNull final ListOfflineRegionsCallback callback) {
+    fileSource.activate();
     listOfflineRegions(fileSource, new ListOfflineRegionsCallback() {
 
       @Override
       public void onList(final OfflineRegion[] offlineRegions) {
         getHandler().post(new Runnable() {
           @Override
           public void run() {
+            fileSource.deactivate();
             callback.onList(offlineRegions);
           }
```

Comment:
Should fileSource.deactivate();
happen after the callback.onList(offlineRegions); ?

Reply:
I rebased osana-offline branch and was not able to see listed offline regions any more.
Reversing the order - fixed my issue

Reply:
> I rebased osana-offline branch and was not able to see listed offline regions any more.
Reversing the order - fixed my issue

startled that this is the case, could you follow up in your branch with reversing?

Reply:
I updated the branch with that change.